pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -813,5 +830,9 @@ func getInitContainerImage(scanSpec *compv1alpha1.ComplianceScanSpec, logger log
 }
 
 func isPlatformScan(instance *compv1alpha1.ComplianceScan) bool {
-	return strings.ToLower(string(instance.Spec.ScanType)) == "platform"
+	return instance.Spec.ScanType == compv1alpha1.ScanTypePlatform
```

Comment:
While the code reads nicer after the change, removing the tolower(string).. construct would regress the fix done in c6ebe30eabac8c36aee67eb4510cdb8b9cb6b0b3

Reply:
why do we need to allow lowercase?

Reply:
answerd via slack